src/components/UnstyledButton/UnstyledButton.tsx

```diff
@@ -6,20 +6,22 @@ import {UnstyledLink} from '../UnstyledLink';
 export interface UnstyledButtonProps {
   /** The content to display inside the button */
   children?: React.ReactNode;
-  /** A destination to link to, rendered in the href attribute of a link */
-  url?: string;
   /** A unique identifier for the button */
   id?: string;
   /** A custom class name to apply styles to button */
   className?: string;
-  /** Disables the button, disallowing merchant interaction */
-  disabled?: boolean;
-  /** Allows the button to submit a form */
-  submit?: boolean;
+  /** A destination to link to, rendered in the href attribute of a link */
+  url?: string;
   /** Forces url to open in a new tab */
   external?: boolean;
   /** Tells the browser to download the url instead of opening it. Provides a hint for the downloaded filename if it is a string value */
   download?: string | boolean;
+  /** Allows the button to submit a form */
+  submit?: boolean;
+  /** Disables the button, disallowing merchant interaction */
+  disabled?: boolean;
+  /** Replaces button text with a spinner while a background action is being performed */
+  loading?: boolean;
```

Comment:
Do you think this one makes sense here? I was debating this too. It does seem like it could be useful functionality to share, but also feels like a visual element that could behave strangely depending how the button is styled.

Reply:
Never mind, I see now this only handles the a11y portion in UnstyledButton. That makes a lot of sense to me.